### PR TITLE
2023 assets fixes

### PR DESCRIPTION
**Pull request by StevenClontz**
Addresses #78

### CODE REVIEW

**Comment by StevenClontz** [comment]
Live view of me just replacing the broken 2023 assets with 2022 versions (may not be perfect but is way closer than what we have now).

**Comment by siwelwerd** [comment]
Looks like you are also removing the QR codes and thumbnails from the videos in a couple subdirectories?

**Comment by StevenClontz** [comment]
Fixed missing qrs, thanks.

I can also confirm that using PreTeXt-CLI 2.0.2 I'm getting the wrong assets with the 2024 preview edition. So there's something probably broken with PreTeXt-CLI or PreTeXt, but we can work around it by adding xml:ids to things which is a good idea anyway.

File: site/2023/generated/youtube/video-50.jpg
Comment (siwelwerd) [inline site/2023/generated/youtube/video-50.jpg:1]: Is it intended to also remove the contents of this `youtube` subdirectory?  I think these are the thumbnail images that go in the print version?

**Review by siwelwerd — CHANGES_REQUESTED** [review]
@StevenClontz : Please clarify whether you intended to also remove the contents of the youtube subdirectory as part of this fix.

File: site/2023/generated/youtube/video-50.jpg
Comment (StevenClontz) [reply site/2023/generated/youtube/video-50.jpg:1]: not intended, but doesn't break anything as they're only used for the print version (same with QRs). Will need a better fix, but I think deploying this as-is is an overall improvement.

Comment (StevenClontz) [reply site/2023/generated/youtube/video-50.jpg:1]: unless you see anything else, I'm running a full rebuild of 2024 preview and deploying the hotfixed 2023 assets in the next few minutes.

**Review by siwelwerd — APPROVED** [review]

**Pull request merged**